dbms/src/Flash/tests/gtest_compute_server.cpp

```diff
@@ -39,6 +39,7 @@ extern const char exception_before_mpp_root_task_run[];
 extern const char exception_during_mpp_non_root_task_run[];
 extern const char exception_during_mpp_root_task_run[];
 extern const char exception_during_query_run[];
+extern const char force_agg_prefetch[];
```

Comment:
todo: why force_agg_prefetch in `ComputeServerRunner.testErrorMessage` will hang

Reply:
from @windtalker : `setCancelTest()` will make table scan output infinite output data, so this case will run forever if this case doesn't report error